src/cursor.rs

```diff
@@ -353,18 +188,15 @@ impl Command for ResetPos {
 
     #[cfg(windows)]
     fn execute_winapi(&self) -> Result<()> {
-        WinApiCursor::new().restore_position()
+        sys::restore_position()
     }
 }
 
-/// A command to hide the cursor.
+/// A command that hides the terminal cursor indicator.
```

Comment:
Can we remove the `indicator` word? We don't use it anywhere in the docs. It was added in this PR for just two commands `Hide` & `Show`.